assets/dev/scss/frontend/widgets/button.scss

```diff
@@ -0,0 +1,48 @@
+.elementor-widget-button {
+
+    .elementor-button{
+
+        span {
+            align-self: center;
+        }
+
+        &-icon {
+            
+            i {
+                color: var(--button-icon-color, inherit);
+                
+            }
+            
+            svg {
+                fill: var(--button-icon-color, inherit);
+                
+            }
+
+            svg:hover {
+                fill: var(--button-icon-color, inherit);
+            }
+        }
+
+        .e-font-icon-svg {
+            font-size: var(--n-button-icon-size);
+        }
+       
+    }
+
+    .elementor-button:hover{
+
+        .elementor-button-icon {
+
+            i {
+                color: var(--button-icon-color-hover);
+                transition: all 0.3s;
```

Comment:
The button hover effect has a transition duration already. Can we use that one? 
We shouldn't add a separate styling for this.

Can you please test this how the button hover transition durations works and how if we need any additional styling.

Reply:
the color of the icon was loading quickly and was faster than the color of the button on hover, so I took the transition of the button and I used the same values for the icon

Reply:
Can you make sure that you use the value from the control and the hardcoded value?

Reply:
@Mchris-hub I will need feedback here.